paparazzi-gradle-plugin/src/test/projects/multiplatform-plugin-without-android/build.gradle

```diff
@@ -0,0 +1,21 @@
+plugins {
+  id 'org.jetbrains.kotlin.multiplatform'
+  id 'com.android.library'
```

Comment:
Trying to understand if this is a typo, shouldn't this project be without Android?

Reply:
the difference is the absence of:

```
kotlin {
  android()
}
```

as in the other project test